src/python/WMCore/ACDC/CouchFileset.py

```diff
@@ -99,9 +99,22 @@ def add(self, files, mask = None):
         _add_
 
         Add files to this fileset
+
+        Note: if job was lumi based splitted, then we do not have
+        reliable events information. If job was event based splitted,
+        then we do not have reliable lumi information.
         """
         filteredFiles = []
         if mask:
+            for f in files:
+                # There is no LastEvent for last job of a file
+                if mask['LastEvent'] and mask['FirstEvent']:
+                    f['events'] = mask['LastEvent'] - mask['FirstEvent']
```

Comment:
Actually, this code is only used to create the acdc docs (by ErrorHandler) and it's used to calculate the number of events we need to recover, example from my logs:
```
mask {'LastRun': 1L, 'FirstRun': 1L, 'inclusivemask': True, 'runAndLumis': {1L: [[649L, 653L]]}, 'LastEvent': 163001L, 'FirstEvent': 162001L, 'LastLumi': 653L, 'FirstLumi': 649L} for job id 527
```
So if we ( + 1), in the end we will be producing 1 event more than what it should. Does it make sense for you?